config/initializers/mime_types.rb

```diff
@@ -4,9 +4,16 @@
 # Mime::Type.register "text/richtext", :rtf
 
 Mime::Type.register "application/msword", :doc
+Mime::Type.register "application/msword", :dot
+Mime::Type.register "application/rdf+xml", :rdf
 Mime::Type.register "application/vnd.ms-excel", :xls
+Mime::Type.register "application/vnd.ms-excel", :xlt
+Mime::Type.register "application/vnd.ms-excel.sheet.macroEnabled.12", :xlsm
+Mime::Type.register "application/vnd.ms-powerpoint", :ppt
 Mime::Type.register "application/vnd.oasis.opendocument.spreadsheet", :ods
 Mime::Type.register "application/vnd.oasis.opendocument.text", :odt
+Mime::Type.register "application/vnd.openxmlformats-officedocument.present", :pptx
 Mime::Type.register "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet", :xlsx
 Mime::Type.register "application/vnd.openxmlformats-officedocument.wordprocessingml.document", :docx
+Mime::Type.register "text/plain; charset=utf-8", :txt
```

Comment:
Will this mean there will be a change in behaviour for Mainstream assets with a `txt` file extension?

Is this intentional? If so, I think it would be useful to highlight this in the commit note and explain what the content type used to be for this file extension and why we're happy to change it.